### PR TITLE
Tidy up course meta data on mobile

### PR DESCRIPTION
**Pull request by dankmitchell**
### Context
https://trello.com/c/Ng44W5dv/88-clearer-metadata-mobile-styling-on-course-details-pages

### Changes proposed in this pull request
- Add semicolon
- Add margin-bottom for mobile

### Guidance to review
Any course page on mobile

# After
localhost_5000_course_0_l 2 subjects 2 funding 15 pgce false qts false fulltime false parttime false iphone 6_7_8 1

### CODE REVIEW

File: src/Assets/Styles/patterns/_definition-list.scss [hunk 1]
```diff
@@ -15,11 +15,12 @@
 
 %govuk-list--description > dd {
   @include govuk-font($size: 19, $weight: normal);
+  margin: 0 0 10px 0;
```
Comment (fofr) [inline src/Assets/Styles/patterns/_definition-list.scss]: `margin-bottom: govuk-spacing(2)`?

File: src/Views/Course/Index.cshtml [hunk 2]
```diff
@@ -22,10 +22,10 @@
                 <dl class="govuk-list--description">
                     @if (Model.Course.ProviderLocation != null)
                     {
-                        <dt>Location</dt>
+                        <dt><span class="govuk-list--description__label">Location</span></dt>
```
Comment (fofr) [inline src/Views/Course/Index.cshtml]: Is the span and class needed?

Comment (dankmitchell) [reply src/Views/Course/Index.cshtml]: This is to cleanly add the semicolon.

File: src/Views/Course/Index.cshtml [hunk 3]
```diff
@@ -103,8 +103,7 @@
                         }
                         @if (Model.HasSection("entry requirements"))
                         {
-                            <li><a href="#section-entry">Requirements</a></li>
-                        }
+                            <li><a href="#section-entry">Requirements</a></li>                        }
```
Comment (fofr) [inline src/Views/Course/Index.cshtml]: Is this change intentional?
